specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-07-01/managedClusters.json

```diff
@@ -4597,13 +4600,15 @@
       "type": "string",
       "enum": [
         "Ubuntu",
-        "CBLMariner"
+        "CBLMariner",
+        "Windows2019",
+        "Windows2022"
       ],
       "x-ms-enum": {
         "name": "OSSKU",
         "modelAsString": true
       },
-      "description": "Specifies an OS SKU. This value must not be specified if OSType is Windows."
+      "description": "Specifies the OS SKU used by the agent pool. If not specified, the default is Ubuntu if OSType=Linux or Windows2019 if OSType=Windows. And the default Windows OSSKU will be changed to Windows2022 after Windows2019 is deprecated."
```

Comment:
Minor: The last sentence I think should just be: "The default Windows OSSKU will be changed to Windows2022 after Windows2019 is deprecated."

Reply:
By the way, do we backfill this property for old clusters? Say I created a Windows node pool already, and now I am using the new API. What value do I see when I do a GET on my ManagedCluster with the new API? I should see `Windows2019` right?

Reply:
Thank you @matthchr .
1. As discussed ```The default is Windows2022 when Kubernetes >= 1.25```, so add this information into the description directly. Is it suitable?
2. Yes. We have the plan that Regional Looper backfills OSSKU for Windows agent pools.